src/radical/pilot/configs/resource_purdue.json

```diff
@@ -0,0 +1,88 @@
+        "pre_bootstrap_0"             :["module load anaconda"],
+        "default_remote_workdir"      : "$PROJECT",
+        "system_architecture"         : {"exclusive": true}
+    },
+
+    "anvil_gpu":
+    {
+        "description"                 : "Heterogeneous Linux cluster located at Purdue university",
+        "notes"                       : "anvil resource name is for Anvil GPUs partitions",
+        "default_schema"              : "local",
+        "schemas"                     : {
+            "local"                   : {
+                "job_manager_endpoint": "slurm://anvil.purdue.edu/",
+                "filesystem_endpoint" : "file://anvil.purdue.edu/"
+            },
+            "ssh"                     : {
+                "job_manager_endpoint": "slurm+ssh://anvil.purdue.edu/",
+                "filesystem_endpoint" : "sftp://anvil.purdue.edu/"
+            }
+        },
+        "default_queue"               : "gpu",
+        "resource_manager"            : "SLURM",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order": ["SRUN", "MPIRUN"],
+                                         "SRUN" : {},
+                                         "MPIRUN": {}
+                                        },
+        "pre_bootstrap_0"             :["module load anaconda"],
+        "default_remote_workdir"      : "$PROJECT",
+        "system_architecture"         : {"exclusive": true}
+    },
+
+    "anvil_mem":
+    {
+        "description"                 : "Heterogeneous Linux cluster located at Purdue university",
+        "notes"                       : "anvil resource name is for Anvil high memory partitions",
+        "default_schema"              : "local",
+        "schemas"                     : {
+            "local"                   : {
+                "job_manager_endpoint": "slurm://anvil.purdue.edu/",
+                "filesystem_endpoint" : "file://anvil.purdue.edu/"
+            },
+            "ssh"                     : {
+                "job_manager_endpoint": "slurm+ssh://anvil.purdue.edu/",
+                "filesystem_endpoint" : "sftp://anvil.purdue.edu/"
+            }
+        },
+        "default_queue"               : "highmem",
+        "resource_manager"            : "SLURM",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order": ["SRUN", "MPIRUN"],
+                                         "SRUN" : {},
+                                         "MPIRUN": {}
+                                        },
+        "pre_bootstrap_0"             :["module load anaconda"],
+        "default_remote_workdir"      : "$PROJECT",
+        "system_architecture"         : {"exclusive": true}
+    }
```

Comment:
missed `"cores_per_node"              : 128,`

maybe we also can add here `"mem_per_node": 1000000` just to differentiate from the regular configuration? @andre-merzky (I don't have strong opinion on this, just suggesting, but ok to skip it)